data/human/deep missions.txt

```diff
@@ -3387,7 +3387,7 @@ mission "Mani Refit"
 	on offer
 		conversation
 			`Your inbox blinks with a new message from the Transport Union:`
-			`	"The fleet is ready for its inspirational journey. Please return to Mani.`
+			`	"The fleet is ready for its inspirational journey. Please return to Mani."`
 			`		-Jayden, Mani Administrator, Transport Union"`
```

Comment:
Remove the quotation marks from both of these lines. The opening quotation mark on this first line is closed by the one on the second line, but this is a text message, not a spoken quote, so it shouldn't be quoted.

Reply:
In this case, how do you identify what is being quoted as part of that text message? Indentation? If so, shouldn't it be indented further, for cases (or only in those cases?), where the line above the quote is also indented, due to it not being the first line of the conversation?

Reply:
This is the style of more "formal" written messages that the player is reading. 
https://github.com/endless-sky/endless-sky/blob/9aaa25867640a24f2d5450698ba682feacb1526f/data/human/free%20worlds%200%20prologue.txt#L2070-L2079

Reply:
Alright, I am removing the quotes for those cases.
About indentation, in this case, there is only two levels before the signatures, should this be fixed in this PR?